src/lightning_app/__version__.py

```diff
@@ -1 +1,18 @@
-version = "1.9.0dev"
+# in case you have not installed the package and have only set python path to this source
+
+import os
+
+_PATH_PKG = os.path.dirname(__file__)
+_PATH_VER = os.path.join(_PATH_PKG, "version.info")
+version: str
+
+if os.path.isfile(_PATH_VER):
+    with open(_PATH_VER, encoding="utf-8") as fo:
+        version = fo.readlines()[0].strip()
+else:
+    try:
+        import subprocess
+
+        version = subprocess.check_output(["git", "rev-parse", "--short", "HEAD"]).decode("ascii").strip()
+    except Exception:
+        version = "unknown"
```

Comment:
this is what I meant with my previous comment.

```suggestion
with open(_PATH_VER, encoding="utf-8") as fo:
    version = fo.read().strip()
```

Since `version.info` will exist, we can always just load it

Reply:
But it will exist only if you install, not when you just clone and set `PYTHONPATH` to this folder so for example developers...

Reply:
That is correct, also in this would allow us to understand if someone is using something else then version installed from PyPI, as installed from source can be easily version in range 100+ commits